### DISCUSSION TITLE
PDE is not satisfied in Inverse Problem

### DISCUSSION THREAD

**Discussion by ppandit95** [lululxvi/deepxde#836] [Q&A]
Dear Dr. Lu

I am trying to run inverse problem in order to determine the coefficients in my PDE,but after the training process what I could see is that the output is mapped accurately(where training data is passed with PointSetBC) with respect to input but my parameters still do not satisfy the PDE and I am getting pretty large residual. A guess from my side is that the order of my PDE error term is less than the order of my BC error term due to which the neural network is trying to minimize the BC error preferably as compared to PDE error term . Alternatively, I would like to know your comments on the above issue.

Many Thanks
Pushkar

**Comment by lululxvi** [comment]
It could be the reason.

**Reply by ppandit95** [reply]
Ohh thanks @lululxvi for commenting on that and I guess I need to tryout iterations with either different weights or implement hPINN as demonstrated in one of the research paper on inverse design

**Reply by haison19952013** [reply]
In my experience, you just let the model run a few iterations to observe the order of magnitude of all loss functions first. Then stop training and adjust the weights to balance those loss functions and retrain.

**Comment by ZSTanone** [comment]
Hi，have you tried to increase the loss weight of PDEs ？can it work？

**Reply by ppandit95** [reply]
Thanks @ZSTanone for suggesting that as it is working now......
